core/src/scene/scene.h

```diff
@@ -28,12 +28,21 @@ struct Stops;
  * Scene is a singleton containing the styles, lighting, and interactions defining a map scene
  */
 
+#define COMPONENT_PATH_DELIMITER '.'
```

Comment:
Putting `#define` directives in headers is somewhat dangerous because they have global scope wherever they end up being included. It doesn't look like this is used in the header, so can we put it in the implementation file instead?